### ISSUE TITLE
Major Error: [Exception] Farmer - Timeout of 30000 ms exceeded

### ISSUE THREAD

**Issue by throwaway45632** [Tom60chat/Microsoft-Rewards-Farmer-Sharp#181]
Microsoft added a reward card for scanning receipts in the Microsoft Start app, you have to scan 6 receipts to get $3 plus 1,000 points. If you can't complete the task then the farmer gets stuck on this card and it can't progress to doing Bing searches.

If would be great it the maker of the program could reverse of the order of Bing searches and reward cards so the searches will happen first and the cards will be done after.

![Screenshot 2023-06-14 110202](https://github.com/Tom60chat/Microsoft-Rewards-Farmer-Sharp/assets/110213411/7ff9883a-fe0f-4d17-a5f9-16662fac5f05)

**Comment by throwaway45632** [comment]
Another way to solve this is to add a flag to skip cards and just do searches

**Comment by vhbjbjbj** [comment]
I agree that. If farmer can just skip the card after 30s it will be quite easy to use. Since the search task is the most important

**Comment by RandyPlayzMC** [comment]
Same problem here

**Comment by throwaway45632** [comment]
I found a short term solution until this deal goes away or the developer can make a specific fix. Set the -nh argument to disable headless mode, wait until the browser goes to the Microsoft Rewards page and shows the "Earn 1,000 points" card. Close the card and click on random cards that you have already completed so that the search window will open, and at some point the random search feature will begin. Once it completes you will get all your daily points.

**Comment by vhbjbjbj** [comment]
> I found a short term solution until this deal goes away or the developer can make a specific fix. Set the -nh argument to disable headless mode, wait until the browser goes to the Microsoft Rewards page and shows the "Earn 1,000 points" card. Close the card and click on random cards that you have already completed so that the search window will open, and at some point the random search feature will begin. Once it completes you will get all your daily points.

now I'm using old edition. The 0.9.2 runs well, but u have to check cards after the main search
